algoliasearch/algoliasearch.go

```diff
@@ -172,6 +172,55 @@ type Client interface {
 	// also accepts extra RequestOptions.
 	MultipleQueriesWithRequestOptions(queries []IndexedQuery, strategy string, opts *RequestOptions) (res []MultipleQueryRes, err error)
 
+	// List all the cluster available for the application
```

Comment:
`s/all the cluster available/all available clusters` and missing dot at the end.